### PR TITLE
Create footer section with contact form

### PR DESCRIPTION
**Pull request by Drod86**
<h2>In this commit we:</h2>

-Added semantic HTML markup for the footer section including a form
-Added styles for the mobile version of the footer and form
-Added HTML5 form validation attributes
-Connected form action to Formspree.io

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -251,6 +253,87 @@ <h3>Skills</h3>
...
+      <footer>
+        <section class="contact">
+          <h2>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+          <form
+            action="https://formspree.io/f/daniel.r.rodriguez86@gmail.com"
+            method="POST"
+          >
```
Comment (Ibizugbe) [inline index.html:265]: Nice job so far👍. I noticed when I try to submit a message, I get this error below🥲.

image

kindly fix this as it is a requirement that our form is able to collect data by using the [Formspree service](https://formspree.io/html)😉. Kindly set up your `formspre` account and add the right id to the link👍.

image

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Team👋👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Github flow is used👍.
- [x] input fields are wrapped in a single form element👍.
- [x] The associated place holder texts are present👍.
- [x] Input fields have the correct `type` attributes👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Drod86) [reply index.html]: Thank you @Ibizugbe for pointing this out. 
- I have corrected the form action URL.

**Review by shahadat3669 — APPROVED** [review]
Hi @Drod86, @AndresEspi, @montsecruz9 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Good PR Title
✅ Good PR description
✅ Good use of GitHub Flows

## Optional suggestions
You did everything as required by the previous reviewer. But to improve your web application, you can consider the followings:
- [Optional] Great UI! 👍 However, there is a small issue with the vertical scrolling on the page. As per the design, there should be no vertical scrollbar visible. To fix this, you need to ensure that the content on the page fits within the viewport without requiring scrolling.  Please review the content, layout, and sizing of the elements to ensure they are properly adjusted to eliminate the vertical scrolling. By addressing this issue, you will achieve a more polished and seamless user experience. Keep up the good work! 💯

[Daniel-Rodriguez-Developer-Portfolio.webm](https://github.com/Drod86/My_Portfolio/assets/55840999/989185e2-d7b6-453b-8b04-4f2829fde4ec)

- [Optional] Well done on adding the 'Future Features' section heading!👍 However, it would be good to add ["future features"](https://github.com/microverseinc/readme-template#-future-features-) and include 1-3 potential features in this section. Remember in this section, you should outline your plans for the future development of the project. Include details about the features, functionalities, or improvements you intend to add. It should demonstrate your vision and give readers an idea of what to expect. Keep up the good work!

Please include 1-3 potential fractures in this section.

| Your | Professional |
| --- | --- |
| image | image |

- [Optional] I noticed that there are some section headings listed in the "Table of Contents" that are not available in your README file. I have marked these headings in red in the image. To maintain consistency and avoid confusion, it would be best to remove these headings from the "Table of Contents" if the corresponding sections are not present in your README file.

image



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**